optax/transforms/_adding_test.py

```diff
@@ -91,6 +91,14 @@ def test_add_noise_has_correct_variance_scaling(self):
 
       chex.assert_trees_all_close(updates_i, updates_i_rescaled, rtol=1e-4)
 
+def test_none_argument():
```

Comment:
Add an indent (the test must be part of the class AddingTest) and add a self argument to the function.